### PR TITLE
#15 Add option to include version

### PR DESCRIPTION
**Pull request by radtriste**
This would be added as a comment

https://github.com/ppalaga/rpkgtests-maven-plugin/issues/15

### CODE REVIEW

File: src/main/java/org/l2x6/rpkgtests/CreateTestJarsXmlMojo.java [hunk 1]
```diff
@@ -133,6 +136,10 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             final Marshaller m = ctx.createMarshaller();
             m.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
             m.marshal(new Gas(gas), w);
+            if (includeVersion) {
+                w.newLine();
+                w.write("<!-- " + mavenSession.getCurrentProject().getVersion() + "-->");
```
Comment (ppalaga) [inline src/main/java/org/l2x6/rpkgtests/CreateTestJarsXmlMojo.java]: What's wrong with having `<version>` as a regular XML element?

**Comment by ppalaga** [comment]
+1 for the overall idea.

File: src/main/java/org/l2x6/rpkgtests/CreateTestJarsXmlMojo.java (hunk 1 above)
Comment (radtriste) [reply src/main/java/org/l2x6/rpkgtests/CreateTestJarsXmlMojo.java]: well nothing specific. I did not want to break the parsing.
But I can add it as a simple tag

Comment (radtriste) [reply src/main/java/org/l2x6/rpkgtests/CreateTestJarsXmlMojo.java]: btw, where would I include it @ppalaga ?
Current XML is 
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<testArtifacts>
</testArtifacts>
```
I can hardly include the `version` tag in the `testArtifacts`
I would need to create a root element or ?

Comment (ppalaga) [reply src/main/java/org/l2x6/rpkgtests/CreateTestJarsXmlMojo.java]: I'd vote for having `<version>` tags per `<testArtifact>` which we need elsewhere anyway.

Comment (radtriste) [reply src/main/java/org/l2x6/rpkgtests/CreateTestJarsXmlMojo.java]: I see now

**Comment by radtriste** [comment]
@ppalaga all updated. Please let me know what you think

**Review by ppalaga — APPROVED** [review]
Thanks @radtriste !

**Comment by ppalaga** [comment]
Should I release like this @radtriste?

**Comment by ppalaga** [comment]
I'd say this change would be for a major version bump.

**Comment by radtriste** [comment]
> Should I release like this @radtriste?

I would love it ! :D

**Comment by ppalaga** [comment]
I have released 1.0.0. It should be synced to Maven Central within an hour or so.

**Comment by radtriste** [comment]
Thank you !

**Pull request merged**